Modules/Devices/test/Devices.API.Tests.Integration/Models/IdentityResponse.cs

```diff
@@ -0,0 +1,10 @@
+﻿using Devices.API.Tests.Integration.Models;
+using Newtonsoft.Json;
+
+namespace Challenges.API.Tests.Integration.Models;
```

Comment:
Namespace... Please execute a find&replace on the whole project to find all occurences.